Acceleratio.XCellKit/ChartExport/Chart/ChartPropertiesSetup.cs

```diff
@@ -3,58 +3,26 @@
 using DocumentFormat.OpenXml.Drawing.Charts;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using DocumentFormat.OpenXml.Drawing.Spreadsheet;
 using DocumentFormat.OpenXml.Packaging;
+using Boolean = DocumentFormat.OpenXml.Office2013.Drawing.ChartStyle.Boolean;
 using Chart = DocumentFormat.OpenXml.Drawing.Charts.Chart;
 
 namespace Acceleratio.XCellKit
 {
     internal abstract class ChartPropertiesSetup
     {
-        /// <summary>
-        /// Set chart title
-        /// </summary>
-        public virtual string Title { get; set; } = "";
-
-        /// <summary>
-        /// Set X Axis Title.
-        /// </summary>
-        public virtual string AxisXTitle { get; set; } = "";
-
-        /// <summary>
-        /// Set Y Axis Title
-        /// </summary>
-        public virtual string AxisYTitle { get; set; } = "";
-
-        /// <summary>
-        /// Set chart Height
-        /// </summary>
-        public virtual int Height { get; set; }
+#region Properties
 
-        /// <summary>
-        /// Set chart Width
-        /// </summary>
-        public virtual int Width { get; set; }
+        public virtual BaseChartProperties ChartProperties { get; set; } = new BaseChartProperties();
 
-        /// <summary>
-        /// Set color for each series.
-        /// </summary>
-        public virtual List<string> SeriesColor { get; set; }
+        private bool isArgumentDate { get; set; } = false;
 
-        /// <summary>
-        /// Set if Legend is visible
-        /// </summary>
-        public virtual bool Legend { get; set; } = true;
+        private int dataCount { get; set; } = 0;
```

Comment:
_name for privates?

Reply:
Sure, I usually put lowerCase for privates... _lowerCase then?

Reply:
oh, sorry. wasn't paying attention. These are properties, so it's okay as is.